index.html

```diff
@@ -55,97 +56,7 @@ <h2 class="label">LET’S CONNECT</h2>
-            <div class="content">
-                <h2 class="project-title">Multi-Post Stories</h2>
-                <div class="stats">
-                    <h3 class="client">CANOPY</h3>
-                    <div class="dot"></div>
-                    <h3 class="myrole">Back End Dev</h3>
-                    <div class="dot"></div>
-                    <h3 class="year">2015</h3>
-                </div>
-                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
-                    required.
-                </p>
-                <ul class="languages">
-                    <li class="lang">html</li>
-                    <li class="lang">css</li>
-                    <li class="lang">javaScript</li>
-                </ul>
-                <button class="details-btn" type="button">See Project</button>
-            </div>
-        </div>
-        <div class="work-card">
-            <img src="images/project3.svg" alt="Tonic" />
-            <div class="content">
-                <h2 class="project-title">Tonic</h2>
-                <div class="stats">
-                    <h3 class="client">CANOPY</h3>
-                    <div class="dot"></div>
-                    <h3 class="myrole">Back End Dev</h3>
-                    <div class="dot"></div>
-                    <h3 class="year">2015</h3>
-                </div>
-                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
-                    required.
-                </p>
-                <ul class="languages">
-                    <li class="lang">html</li>
-                    <li class="lang">css</li>
-                    <li class="lang">javaScript</li>
-                </ul>
-                <button class="details-btn" type="button">See Project</button>
-            </div>
-        </div>
-        <div class="work-card">
-            <img src="images/project4.svg" alt="Multi-Post Stories" />
-            <div class="content">
-                <h2 class="project-title">Multi-Post Stories</h2>
-                <div class="stats">
-                    <h3 class="client">CANOPY</h3>
-                    <div class="dot"></div>
-                    <h3 class="myrole">Back End Dev</h3>
-                    <div class="dot"></div>
-                    <h3 class="year">2015</h3>
-                </div>
-                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
-                    required.
-                </p>
-                <ul class="languages">
-                    <li class="lang">html</li>
-                    <li class="lang">css</li>
-                    <li class="lang">javaScript</li>
-                </ul>
-                <button class="details-btn" type="button">See Project</button>
-            </div>
-        </div>
-    </section>
-
+    <section class="works" id="Portoflio"></section>
```

Comment:
- [ ] You have done a great job working on popup window. I would only recommend to hide the body, so that users could see only the popup window. Currently, I can scroll the window and see the body content. 
- [ ] I recommend making the popup window smaller as provided in figma design, this way picture takes less space and users can see more popup content without having to scroll down:

Your design  | Figma design
------------- | -------------
<img width="692" alt="Screenshot 2022-03-02 at 12 36 19" src="https://user-images.githubusercontent.com/77038610/156325241-9aa8b6e0-2437-4ce5-a0b2-8f419948ca73.png"> | <img width="904" alt="Screenshot 2022-03-02 at 12 35 14" src="https://user-images.githubusercontent.com/77038610/156325043-f28a71b4-19b4-4e20-a308-5a6a2c0662ed.png">